dlt/load/bigquery/client.py

```diff
@@ -62,7 +62,7 @@ def open_connection(self) -> None:
             credentials = None
         else:
             credentials = service_account.Credentials.from_service_account_info(self.C.as_credentials())
-        self._client = bigquery.Client(self.C.PROJECT_ID, credentials=credentials)
+        self._client = bigquery.Client(self.C.PROJECT_ID, credentials=credentials, location=self.C.LOCATION)
```

Comment:
can you do an experiment?
line: 204
```
return BigQueryLoadJob(
                JobClientBase.get_file_name_from_file_path(file_path),
                self._create_load_job(table["name"], table["write_disposition"], file_path),
                self.CREDENTIALS
            )
            
```

replace self.CREDENTIALS with `self.sql_client.native_connection()`

and see if all works. then we are sure the same client contexts used in other places is passed to the load job

Reply:
there are 2 places, start_file_load, and restore_file_load, should I just do the restore or both?